### PR TITLE
NEPT-2228: Improve access_callback on og pages.

### PR DESCRIPTION
**Pull request by drishu**
### Description

Add access control check to OG content pages.

### Change log

- Added:
    - Add access control to menu entries like community/%group_name/%content_type
    - Behat tests for access control of og content

### Commands

drush cc all

### CODE REVIEW

File: profiles/common/modules/features/multisite_forum_community/multisite_forum_community.info [hunk 1]
```diff
@@ -2,7 +2,10 @@ name = Multisite Forum Community
 description = Community feature for Multisite Forum suite.
 core = 7.x
 package = Multisite OG Features
+dependencies[] = features
+dependencies[] = field_group
 dependencies[] = multisite_forum_core
```
Comment (kameleon69) [inline profiles/common/modules/features/multisite_forum_community/multisite_forum_community.info:5]: You shouldn't have these dependencies on this feature

File: profiles/common/modules/features/multisite_forum_community/multisite_forum_community.info [hunk 2]
```diff
@@ -20,3 +23,4 @@ features[og_features_permission][] = node:community:update own discussion conten
 features[views_view][] = forums_discussions
 features[views_view][] = forums_list
 files[] = tests/multisite_forum_community.test
+php = 5.2.4
```
Comment (kameleon69) [inline profiles/common/modules/features/multisite_forum_community/multisite_forum_community.info]: we are on PHP 5.6

File: profiles/common/modules/features/multisite_forum_community/multisite_forum_community.install [hunk 3]
```diff
@@ -11,13 +11,14 @@
...
-  // Activate provider 'og_context_negotiation_group_context'
-  // in og context configuration.
+  // Activate provider 'og_context_negotiation_group_context' in
+  // og context configuration.
   $defaults = array('url' => -5, 'node' => -4);
   $og_context_negotiation_group_context = variable_get('og_context_negotiation_group_context', $defaults);
   $og_context_negotiation_group_context['multisite_forum_community'] = array();
   variable_set('og_context_negotiation_group_context', $og_context_negotiation_group_context);
 
+  // Add multilinual forum to ogcontexte negocation (not.
```
Comment (kameleon69) [inline profiles/common/modules/features/multisite_forum_community/multisite_forum_community.install]: typo here

**Review by kameleon69 — CHANGES_REQUESTED** [review]

**Comment by Fefaine** [comment]
SEE TICKET

**Pull request closed without merging**